banyan-cli/src/sqlite.rs

```diff
@@ -3,17 +3,25 @@ use anyhow::{anyhow, Result};
 use banyan::store::{BlockWriter, ReadOnlyStore};
 use ipfs_sqlite_block_store::{BlockStore, Config, OwnedBlock};
 use libipld::Cid;
-use std::sync::{Arc, Mutex};
+use std::{
+    path::Path,
+    sync::{Arc, Mutex},
+};
 
 use crate::tags::Sha256Digest;
 
 pub struct SqliteStore(Arc<Mutex<BlockStore>>);
 
 impl SqliteStore {
+    #[allow(dead_code)]
     pub fn memory() -> anyhow::Result<Self> {
         let store = BlockStore::memory(Config::default())?;
         Ok(SqliteStore(Arc::new(Mutex::new(store))))
     }
+    pub fn new(path: impl AsRef<Path>) -> anyhow::Result<Self> {
+        let store = BlockStore::open(path, Config::default())?;
+        Ok(SqliteStore(Arc::new(Mutex::new(store))))
+    }
```

Comment:
i'd either remove `memory()` or do
```suggestion
    pub fn new(store: BlockStore) -> Self {
        SqliteStore(Arc::new(Mutex::new(store)))
    }
```